nixos/modules/config/zram.nix

```diff
@@ -58,12 +91,26 @@ in
         '';
       };
 
+      algorithm = mkOption {
+        default = "zstd";
+        example = "lzo";
+        type = with types; either (enum [ "lzo" "lz4" "zstd" ]) str;
+        description = ''
+          Compression algorithm. <literal>lzo</literal> has good compression,
+          but is slow. <literal>lz4</literal> has bad compression, but is fast.
+          <literal>zstd</literal> is both good compression and fast (kernel 4.18+).
```

Comment:
There could be a warning generated if the kernel is too old (config.boot.kernelPackages.kernel.version)

Reply:
I'd better remove this section. Current kernel is already capable of zstd zram.